sdk/helper/testhelpers/response_validation.go

```diff
@@ -64,3 +66,45 @@ func ValidateResponseData(schema *framework.Response, data map[string]interface{
 
 	return fd.Validate()
 }
+
+// FindResponseSchema is a test helper to extract the response schema from a given framework path / operation
+func FindResponseSchema(t *testing.T, paths []*framework.Path, pathIdx int, operation logical.Operation) *framework.Response {
```

Comment:
Is this not a fuction that we'd also want in the shared dir?

Reply:
It is in a shared dir :)